project/Publish.scala

```diff
@@ -35,10 +35,10 @@ object Publish extends AutoPlugin {
     startYear := Some(2009),
     developers := List(
       Developer(
-        "akka-contributors",
-        "Akka Contributors",
+        "pekko-contributors",
+        "Pekko Contributors",
         "akka.official@gmail.com",
```

Comment:
What is the equivalent email address for `akka.official@gmail.com` ?

Reply:
use `dev@pekko.apache.org`

Reply:
Ignore my previous comment, just now saw on main it is modified